.github/workflows/check-formatting.yml

```diff
@@ -23,15 +23,6 @@ jobs:
       - name: Install dependencies
         run: sudo apt-get update && sudo apt-get install -y build-essential git clang curl libssl-dev llvm libudev-dev make protobuf-compiler pkg-config git-restore-mtime
 
-      - name: Install rust toolchain
```

Comment:
`rustup` is now included in GitHub Runners, so if we have correct stuff in `rust-toolchain.toml` it just works.